fgd/brush/trigger/trigger_momentum_timer_end.fgd

```diff
@@ -0,0 +1,6 @@
+@SolidClass appliesto(MOMENTUM)
+= trigger_momentum_timer_end : "Zone that stops the timer."
```

Comment:
This could be slightly clearer that this is not a "cancel" zone, maybe:
```suggestion
= trigger_momentum_timer_end : "Zone that finishes the timer."
```